cmake_common_scripts/cmake/cmake_tools.cmake

```diff
@@ -1,5 +1,8 @@
+# The code below is licensed under the following license unless explicted stated
```

Comment:
```suggestion
# The code below is licensed under the following license unless stated otherwise,
```

Reply:
Removed. I put the BSD code in another file.